src/StyledFormComponents.js

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components";
+
+const StyledForm = styled.div`
+  height: 400px;
+  padding: 20px;
+`;
+
+const StyledCardHeading = styled.h3`
+  background: #dbd8db;
+  width: 100%;
+  height: 55px;
+  font-size: 30px;
+  font-weight: bold;
+  text-align: center;
+  vertical-align: middle;
+  text-transform: uppercase;
```

Comment:
nit: Should headers be all uppercase? Maybe capitalize is a less aggressive approach!